pillar/heroku/xpro.sls

```diff
@@ -170,6 +169,8 @@ heroku:
     MITOL_DIGITAL_CREDENTIALS_DEEP_LINK_URL: dccrequest://request
     MITOL_DIGITAL_CREDENTIALS_HMAC_SECRET: __vault__::secret-{{ business_unit }}/{{ environment }}/digital-credentials-integration>data>hmac_secret
     MITOL_DIGITAL_CREDENTIALS_VERIFY_SERVICE_BASE_URL: __vault__::secret-{{ business_unit }}/{{ environment }}/digital-credentials-integration>data>sign_and_verify_url
+    MITOL_HUBSPOT_API_PRIVATE_TOKEN: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/hubspot-api-private-token>data>value
```

Comment:
I would change that to `__vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}//hubspot-api-private-token>data>value`

Reply:
We should get out of the practice of using the `{{ env_data.vault_env_path }}` component of the path since we now have distinct Vault clusters for each environment stage. That way the path is easier to figure out and it makes the overall key space a bit flatter.

Reply:
Would this be better?

```
 __vault__::secret-operations/{{ business_unit }}/{{ environment }}/hubspot-api-private-token>data>value
```